DEVELOPER.md

```diff
@@ -23,11 +23,13 @@ Build HTML docs:
 
     $ make docs
 
+Documentation will be generated in `docs/_build/`.
+
 or:
 
     $ python setup.py build_sphinx
 
-Documentation will be generated in `docs/_build/`.
+Documentation will be generated in  `build\sphinx\html`.
```

Comment:
Please use forward-slash, since that works everywhere (backslash only works on windows).